lib/fog/vsphere/requests/request.rb

```diff
@@ -0,0 +1,19 @@
+module Fog
+  module Vsphere
+    module Requests
+      class Request
```

Comment:
This is base class for request, after some flame or fame, I would try to suggest it to `fog-core` as I believe in principle this is a better pattern.